ovos_utils/location.py

```
@@ -0,0 +1,191 @@
+    """
+import ipaddress
+
+def _is_valid_ip(ip):
+    try:
+        ipaddress.ip_address(ip)
+        return True
+    except ValueError:
+        return False
+
+@timed_lru_cache(seconds=600)  # cache results for 10 mins
+def get_ip_geolocation(ip):
```

Ruff

Redefinition of unused `get_ip_geolocation` from line 148

(F811)